### PR TITLE
Create .gitignore

### PR DESCRIPTION
**Pull request by aneallaryea100**
a big trial

### CODE REVIEW

**Review by aneallaryea100 — COMMENTED** [review]
change error in style.css

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
+
+.yml
 node_modules
-test.md
\ No newline at end of file
+test.md
```
Comment (cwaku) [inline .gitignore:4]: Great job on adding this to `.gitignore` file but I realised you pushed the file first before doing this. Please go ahead to remove the `test.md` file from your remote repo.

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @aneallaryea100, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!. Implement these changes and there would be nothing to stop your project from being approved.

## Highlights
- You used a custom README 👍🏾 
- Correct use of Github flow 🥇 
- Good implementation 💯 
- Descriptive commit messages 🥇 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

I was not able to reference them directly so I left the required changes here :
- It is required that your .css file includes one class with styles for your header (make it in your favorite color). I see in your file that you tried to do that but however, but your styles are not reflecting. In your `inddex.html`, you are forgetting to link the stylesheet. Please follow this [tutorial](https://www.bitdegree.org/learn/how-to-link-css-to-html) to learn how to do that. 
[TIP] 👀  : Use `color` instead of `background-color` to change the color of the text.
image

- Here we only need one README file. Please delete [NEWREADME.md](https://github.com/aneallaryea100/HELLO-MICROVERSE/blob/097290e6ec26341873bc66a103621a3146d49c51/NEWREADME.md)
image

- Friendly reminder: You should make sure that your repo looks professional before review.
Please take a step back and read a lesson [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there).
- Always confirm before requesting for review, that the linter result is green. AKA: Linter should report NO errors.
- Lastly, give a short summary of what you did in the PR description. This helps code reviewers know exactly what you did at first glance.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (aneallaryea100) [reply .gitignore:4]: okay. I will do that.

**Pull request merged**